NightscoutUploadKit/DeviceStatus/LoopEnacted.swift

```diff
@@ -13,20 +13,28 @@ public struct LoopEnacted {
     let duration: NSTimeInterval
     let timestamp: NSDate
     let received: Bool
+    let predBGs: [Double]?
     
-    public init(rate: Double, duration: NSTimeInterval, timestamp: NSDate, received: Bool) {
+    public init(rate: Double, duration: NSTimeInterval, timestamp: NSDate, received: Bool, predBGs: [Double]? = nil) {
```

Comment:
Actually, setting the env var DISPLAY_UNITS didn't seem to do anything.  There is a preference on the settings ui that can switch between mmol and mg/dl, and that does seem to work.